src/DistributedLock.Redis/RedisDistributedSemaphore.cs

```diff
@@ -39,6 +39,19 @@ public RedisDistributedSemaphore(RedisKey key, int maxCount, IDatabase database,
     /// </summary>
     public string Name => this.Key.ToString();
 
+    public int GetCurrentCount()
+    {
+        return GetCurrentCountAsync().GetAwaiter().GetResult();
+    }
+    
+    public async ValueTask<int> GetCurrentCountAsync()
+    {
+        var database = _databases[0];
+        var primitive = new RedisSemaphorePrimitive(this.Key, this.MaxCount, this._options.RedLockTimeouts);
+        var acquiredCount = await primitive.GetCurrentCountAsync(database).ConfigureAwait(false);
+        return this.MaxCount - acquiredCount;
```

Comment:
Let's wrap this with `Math.Max(0, ...)`. Because semaphores can be declared with different counts on different machines (not a good thing, but it could happen), it's possible that acquiredCount could be greater than MaxCount. However, `CurrentCount` shouldn't return a negative number in that case.